README.md

```diff
@@ -1,133 +1,111 @@
-# Ankermake M5 Protocol
+# AnkerMake M5 Protocol
 
-Welcome! This repository contains `ankerctl.py`, a command-line interface for
-monitoring, controlling and interfacing with Ankermake M5 3D printers.
+Welcome! This repository contains `ankerctl.py`, a command-line interface for monitoring, controlling and interfacing with AnkerMake M5 3D printers.
 
-NOTE: This software is in early stages, so expect sharp edges and occasional errors!
+**NOTE:** This software is in early stages, so expect sharp edges and occasional errors!
 
-The `ankerctl` program uses [`libflagship`](documentation/libflagship.md), a
-library for communicating with the numerous different protocols required for
-connecting to an Ankermake M5 printer. The `libflagship` library is also maintained
-in this repo, under [`libflagship/`](libflagship/).
+The `ankerctl` program uses [`libflagship`](documentation/libflagship.md), a library for communicating with the numerous different protocols required for connecting to an AnkerMake M5 printer. The `libflagship` library is also maintained in this repo, under [`libflagship/`](libflagship/).
 
-## Current features
+## Features
 
- - Print directly from PrusaSlicer, SuperSlicer
+### Current Features
 
- - Connect to Ankermake M5 and Ankermake APIs without using closed-source Anker
-   software.
+ - Print directly from PrusaSlicer and its derivatives (SuperSlicer, Bamboo Studio, OrcaSlicer, etc.)
 
- - Send raw gcode commands directly to the printer (and see the response)
+ - Connect to AnkerMake M5 and AnkerMake APIs without using closed-source Anker software.
 
- - Low-level access to MQTT, PPPP and HTTPS APIs
+ - Send raw gcode commands directly to the printer (and see the response).
```

Comment:
We cannot send raw gcode *directly* to the printer. We can send it over mqtt, and hope that the printer will listen for it. Also, the responses are truncated to 512 bytes :-/

I know it might seem like a nitpick, but it feels like we are promising the users something we can't do, here.

Reply:
Resolved by removing "directly"